### DISCUSSION TITLE
RocketFuel 4.0 Released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2962] [Show and tell]
We are pleased to inform you that the RocketFuel Theme has been released for Oqtane version 4 and .Net 7. This theme introduces several modifications and tweaks to the CSS, specifically designed to enhance the user experience when switching between Light and Dark modes. These adjustments aim to ensure a smooth and visually appealing transition. We believe that the RocketFuel Theme will bring added value and aesthetics to your website.

**Comment by Azath07** [comment]
i have installed RocketFuel 4.0 theme in Oqtane 4.0.5 but it have some bugs , come the buttons are not clicking when will this bugs will be fixed and is there any ways to fix
![error](https://github.com/oqtane/oqtane.framework/assets/123856484/9ea8546f-d63b-40a0-b739-3002daebdb5a)

**Reply by leigh-pointer** [reply]
@Azath07 you say you have installed 4.0, please install 4.1.0 and retry.  The error does not seem to be originating from the Theme so I would be interested to see the  error log.

**Reply by Azath07** [reply]
![error 2](https://github.com/oqtane/oqtane.framework/assets/123856484/ebf76c5a-5779-4d7d-a866-b3b09f82e660)
@leigh-pointer i have reinstalled and updated into  rocketfuel 4.1.0  Still got the same error and some of the functions are not working properly and the same bug here

**Reply by leigh-pointer** [reply]
@Azath07 please send me the error log. Also check the same functionality with the default Oqtane theme!

**Comment by leigh-pointer** [comment]
@eveyone can someone who uses rocketfuel confirm this issue please?
________________________________
